Gems/Atom/RPI/Code/Source/RPI.Builders/Model/MaterialAssetBuilderComponent.cpp

```diff
@@ -44,7 +45,7 @@ namespace AZ
             if (auto* serialize = azrtti_cast<SerializeContext*>(context))
             {
                 serialize->Class<MaterialAssetDependenciesComponent, Component>()
-                    ->Version(5) // Set materialtype dependency to OrderOnce
+                    ->Version(5) // <<<<< This probably is NOT the version number you want to bump. What you're looking for is MaterialAssetBuilderComponent::Reflect below
```

Comment:
For the future reader without context about this change, it may be worth adding to your already helpful comment a statement about what this component _does_ correspond to